dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -3,18 +3,31 @@
 mkdir -p /etc/frr
 mkdir -p /etc/supervisor/conf.d
 
+MGMT_FRAMEWORK_CONFIG=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["frr_mgmt_framework_config"]'`
```

Comment:
The backticks are outdated. We use $()

@tahmed-dev Tamer, Is it possible to avoid calling sonic-cfggen individually here?

Reply:
Thanks @pavel-shirshov! Yes we could add this to the frr_vars.j similar to what we do with [telemetry](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry_vars.j2). We can then use `jq` utility to extract fields from the json file as done [here](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-sonic-telemetry/telemetry.sh#L15)

Reply:
@pavel-shirshov @tahmed-dev In this case, we need to firstly get the frr_mgmt_framework_config flag, then based on the flag we decide which template files need to be decoded. Looks these two steps all need to call sonic-cfggen. I think one solution is we put flag check in bgpd.conf.j2 and staticd.conf.j2 and add extra steps just parse bfdd.conf.j2 and ospfd.conf.j2 if the frr_mgmt_framework_config flag is true

Reply:
Addressed $() issue

Reply:
@tahmed-dev  In order to select the .j2 files we need to first check whether frr_mgmt_framework_config field is present or not in the config-DB, so, looks like two sonic-cfggen invocations are inevitable, but can we do the following to avoid two sonic-cfggen calls.
MGMT_FRAMEWORK_CONFIG=`sonic-db-cli CONFIG_DB HGET 'DEVICE_METADATA|localhost' "frr_mgmt_framework_config"`

Reply:
@venkatmahalingam I could think of two approaches to this problem:
1. create new template that conditionally include other templates `/usr/local/frr/..` or `/usr/share/sonic/..` based on the value of `frr_mgmt_framework_config`, or
2. generate all the files and then delete/rename files as needed.

I am favoring approach 1.

Reply:
@tahmed-dev We need to generate separate conf file for each daemon (e. g. bgpd, ospfd, staticd, ...), so I think what we could do is adding conditional include on all jinja templates (e. g. bgpd.conf.j2, staticd.conf.j2 ...). And for the daemons that are currently only used by frr_mgmt_framework_config like ospfd and bfdd, we can give an empty conf file if the flag is not set. Is this the way as you suggested?

Reply:
Or I could deleted the generated ospfd.conf and bfdd.conf if frr_mgmt_framework_config flag is not set

Reply:
@tahmed-dev As discussed in meeting, frr_mgmt_framework_config flag will be checked in bgpd.conf.j2 and static.conf.j2 to decide which file need to be included. And bfdd.conf and ospfd.conf will always be generated but will deleted if frr_mgmt_framework_config flag is not set.

Reply:
Addressed